### PR TITLE
Updates to spring basics Codelab with information on usage of local (docker) vs cloud broker

### PR DESCRIPTION
**Pull request by gvensan**
Updated the codelab with section for local broker vs cloud broker wherever applicable and enhanced instructions around application.properties and application.yml file content

### CODE REVIEW

File: markdown/spring-cloud-stream-basics/spring-cloud-stream-basics.md [hunk 1]
```diff
@@ -173,12 +173,16 @@ If you look at the `pom.xml` file you'll see a few important things:
...
 - The "Spring Cloud Stream" and "Spring Cloud Starter Stream Solace" dependencies are included
 
-![Pom File](img/pom.webp)
+![Pom File](img/pom.jpg)
 
 ### Add Messaging Service Connection Info
 
 Open the application config file under `src/main/resources` and enter the properties below substituting the connection information with your messaging services' connect info we got from the Solace Cloud Connect tab in the previous section.
 
+### <strong>A. PubSub+ Broker running locally in Docker</strong>
```
Comment (TamimiGitHub) [inline markdown/spring-cloud-stream-basics/spring-cloud-stream-basics.md:182]: using a `<strong>` would still work for bolding; for future reference you can also use `**` as follows
```
### **A. PubSub+ Broker running locally in Docker**
```

**Comment by TamimiGitHub** [comment]
Looks good to me, giri! thanks for this. I have one minor comment, looks like the bullet points on the last takeaway section is broken. Could you please fix it, re-export, commit, and push to update the PR? Other than that we can merge in after this change

Screen Shot 2021-08-09 at 12 09 46 PM

File: markdown/spring-cloud-stream-basics/spring-cloud-stream-basics.md (hunk 1 above)
Comment (gvensan) [reply markdown/spring-cloud-stream-basics/spring-cloud-stream-basics.md:182]: Sure!

**Comment by gvensan** [comment]
Ok, will fi that.

**Review by TamimiGitHub — APPROVED** [review]
LGTM

**Pull request merged**